Demo_Project/src/Component/ToolBar/ToolBar.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import useStyles from "./ToolBarStyles";
+export default function ToolBar({ propsdata }: { propsdata: any }) {
+  const classes = useStyles();
+  const [yourStyle, setyourStyle] = useState(classes.title);
```

Comment:
as mentioned previously, this isn't the right way to do it, 
take a search around the web